app/src/ui/lib/update-branch.ts

```diff
@@ -24,7 +24,7 @@ export function getMergeOptions(): ReadonlyArray<IDropdownSelectButtonOption> {
     {
       label: 'Rebase',
       description:
-        'The commits from the selected branch will be rebased and added to the current branch.',
+        'The selected branch is applied as the new base commit of the current branch and the excess commits in the current branch are applied on top of the new base.',
```

Comment:
I think this description is harder to understand than the previous one 🤔  Does it add any info I might be missing? 😳

Reply:
I will pull this change out to a new PR, that migh make discussion a bit easier. But granted, the new description is not optimal too :-D

Reply:
Extracted.